app.ts

```diff
@@ -25,6 +26,7 @@ render(app, {
     root: path.join(__dirname, "views"),
     layout: "layout",
 });
+app.use(jsonPlaceholder.routes()).use(jsonPlaceholder.allowedMethods());
```

Comment:
consolidate all the app.use(<route_name>.routes()).use(<route_Name>.allowedMethods) into one single 
app.use( <name>.routes()).use(<name>.allowedMethods)